packages/cli/src/commands/generate/helpers.js

```diff
@@ -125,6 +125,7 @@ export const createYargsForComponentGeneration = ({
         await tasks.run()
       } catch (e) {
         console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
        console.error(c.error(e.message))
```